core/ir/disassemble_shared.c

```diff
@@ -399,6 +399,11 @@ opnd_base_disp_disassemble(char *buf, size_t bufsz, size_t *sofar INOUT, opnd_t
                 print_to_buffer(buf, bufsz, sofar, "-");
             }
         }
+#if defined(RISCV64)
```

Comment:
How about removing the ifdefs and adding DR_DISASM_RISCV instead.  We want to move away from ifdefs in core/ir so we can have one build of drdecode decode any arch (#1684).  This also matches ARM.

Reply:
This is indeed a better way, done in https://github.com/DynamoRIO/dynamorio/pull/6095/commits/09399327a52f14082f4275c07f4d47f3a352f3eb.

Reply:
I also removed the register prefix `%` for RISC-V in above commit.